cs/src/Connections/TunnelRelayTunnelHost.cs

```diff
@@ -295,16 +298,19 @@ private void HostSession_ChannelOpening(object? sender, SshChannelOpeningEventAr
             e.Channel.ChannelType == "forwarded-tcpip")
         {
             // V2 protocol.
+            var relayRequestMessage = e.Channel.OpenMessage
+                .ConvertTo<PortRelayConnectRequestMessage>();
```

Comment:
Same question here. What if the service sends out the old message, will it cause any trouble?

Reply:
It's V2 protocol only.

Note the current V2 websocket subprotocol has the "-dev" suffix. When V2 is stable we'll drop the "-dev" suffix, so any non-updated clients will not connect with an incomplete implementation of V2 protocol.

Reply:
Added comments.